packages/keyring-controller/src/KeyringController.ts

```diff
@@ -2201,9 +2194,6 @@ export class KeyringController extends BaseController<
       // NOTE: Not all keyrings implement this method in a asynchronous-way. Using `await` for
       // non-thenable will still be valid (despite not being really useful). It allows us to cover both
       // cases and allow retro-compatibility too.
-      // FIXME: For some reason, it seems that eslint is complaining about this call being non-thenable
-      // even though it is... For now, we just disable it:
-      // eslint-disable-next-line @typescript-eslint/await-thenable
```

Comment:
Ah, thanks for fixing/removing this one 😄 This drove me crazy last time! Might have been a false-positive and the `eslint` update fixed it!